src/locales/en.json

```diff
@@ -109,9 +109,12 @@
     "location_subprompt_US-LA": "Entire state, or select parish...",
     "location_subprompt_US-AK": "Entire state, or select borough...",
     "location_subprompt_US": "Entire state, or select county...",
-    "location_selector_header": "Step 1: Enter your location",
+    "location_selector_header": "Step 1: Location Risk",
```

Comment:
For this one, I'd prefer to opt for "Enter your location" so it is a clear instruction. 

I can see why you made this change. If you'd prefer, I think it's fine for this header to change when switched to manual data mode.

Whatcha think?